README.md

```diff
@@ -1 +1,132 @@
-# siddhi-io-twitter
\ No newline at end of file
+﻿﻿siddhi-io-twitter
+======================================
+The siddhi-io-twitter extension is an extension to Siddhi which is used to receive event data from Twitter API. It supports json format.
+
+**Prerequisites</a>**
+======================================
+* Go to the https://apps.twitter.com/ and create new App
+* Select the app created in step 1 and go to “permission” tab and select “read&write” permission
+* Go to the “keys and access tokens” tab and generate new access token
+* Collect following value from “keys and access tokens tab"
+  * Consumer key
+  * Consumer Secret
+  * Access Token
+  * Access Token Secret
+
```

Comment:
Include a statement saying update the attributes for the extension with these values